android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java

```diff
@@ -165,6 +184,31 @@ protected void onDestroy() {
         super.onDestroy();
     }
 
+    private void showControls() {
+        Boolean autoHide = pluginOptions.getBoolean(AUTO_HIDE_CONTROLS_OPTION, true);
+
+        if (isAnimatingControls || !autoHide) return;
+
+        isAnimatingControls = true;
```

Comment:
I think this variable shouldn't exist. Rather, you could use `controls.getVisibility()`.